ingest/indexer/service/indexer_streaming_service.go

```diff
@@ -285,3 +291,24 @@ func (s *indexerStreamingService) ListenCommit(ctx context.Context, res abci.Res
 func (s *indexerStreamingService) Stream(wg *sync.WaitGroup) error {
 	return nil
 }
+
+// trackCreatedPoolID tracks the created pool ID.
+// If the pool ID is not found in the event attributes, it logs an error.
+// If the pool ID is found, it parses the pool ID to uint64 and tracks it.
+func (s *indexerStreamingService) trackCreatedPoolID(event abci.Event, blockHeight int64, blockTime time.Time, txHash string) {
```

Comment:
Note to reviewer: I'm passing the creation details (block height, block time, txn hash) along here so that those details can be made available when we publish the pair. Please let me know what you think. Thanks!

Reply:
Small nit: we could reduce the number of parameters by encapsulating block metadata them into a struct